ui/admin/app/routes/scopes/scope/workers/worker/index.js

```diff
@@ -4,5 +4,12 @@
  */
 
 import Route from '@ember/routing/route';
+import { action } from '@ember/object';
 
-export default class ScopesScopeWorkersWorkerIndexRoute extends Route {}
+export default class ScopesScopeWorkersWorkerIndexRoute extends Route {
+  @action
+  cancel(worker) {
+    worker.rollbackAttributes();
+    this.refresh();
+  }
```

Comment:
Is this necessary?  Cancel is a route-action and should bubble up to workers.js.